notebooks/11_temporal_probability_models/index.md

```diff
@@ -0,0 +1,603 @@
+Predict using process model:
+
+$$
+\bar{x}_{k+1}=f(\hat{x}_k,u_k,0) 
+$$
+
+$$
+\bar{P}_{k+1}=J_{f_x}( \hat{x}_k,u_k,0)P_kJ^T_{f_x}( \hat{x}_k,u_k,0)+J_{f_w}( \hat{x}_k,u_k,0)QJ^T_{f_w}( \hat{x}_k,u_k,0)
+$$
+
+where $J_{f_x}(\hat{x}_k,u_k,0)$  is the Jacobian of function $f$ with respect to $x$,  $J_{f_w}(\hat{x}_k,u_k,0)$  is the Jacobian of function f with respect to $w$, both evaluated at  $(\hat{x}_k,u_k,0)$ .
+Update using observation:
+
+$$
+\hat{x}_{k+1}=\bar{x}_{k+1}+K(z_{k+1}−h(\bar{x}_{k+1}))
+$$
+
+$$
+P_{k+1}=\bar{P}_{k+1}−KSK^T
+$$
+
+where the innovation covariance $S$ (here  $z_{k+1}−h(\bar{x}_{k+1})$  is called innovation) and the Kalman gain $K$ are given by
+
+$$
+S=J_h(\bar{x}_{k+1})\bar{P}_{k+1}J^T_h(\bar{x}_{k+1})+R
+$$
+
+$$
+K=\bar{P}_{k+1}J^T_h(\bar{x}_{k+1})S^{−1}
+$$
+
+where  $J_h(\bar{x}_k+1)$  is the Jacobian of function h with respect to x evaluated at  $\bar{x}_{k+1}$ .
+Recursive application of the above equations every instant a new observation is gathered yields an updated estimate for the current robot location and its uncertainty. This recursive nature makes EKF the most computationally efficient algorithm available for robot localization.
+An important prerequisite for EKF-based localization is the ability to associate measurements obtained with specific landmarks present in the environment. Landmarks may be artificial, for example, laser reflectors, or natural geometric features present in the environment such as line segments, corners, or planes. In many cases, the observation itself does not contain any information as to which particular landmark is being observed. Data association is the process in which a decision is made as to the correspondence between an observation from the sensor and a particular landmark. Data association is critical to the operation of an EKF-based localizer, as catastrophic failure may result if data association decisions are incorrect.
+EKF relies on approximating the nonlinear motion and observation models using linear equations and that the sensor noises can be approximated using Gaussian distributions. These are reasonable assumptions under many practical conditions and therefore EKF is the obvious choice for solving the robot localization problem when the map of the environment consists of clearly identifiable landmarks.
+
+Figure 2 shows the result of EKF localization for the simple problem given in Figure 1. The ground truth of the robot poses and the estimated robot poses are shown in red and blue, respectively. The 95% confidence ellipses obtained from the covariance matrices in the EKF estimation process are also shown in the figure.
+| ![Figure 1](https://onlinelibrary.wiley.com/cms/asset/7c427b65-cce0-4158-be66-717b00a419ac/nfg002.gif) | 
+|:--:| 
+| *Figure 1* |
+
+![Figure 2](https://onlinelibrary.wiley.com/cms/asset/beb02148-6fb8-4080-b76e-6770ad04f8ff/nfg004.gif) | 
+|:--:| 
+| *Figure 2* |
+
+# Dynamic Bayes Nets
```

Comment:
review your english since this is pretty much formal... I will mention a couple of examples in this section